perllib/FixMyStreet/Cobrand/Zurich.pm

```diff
@@ -619,9 +626,7 @@ sub admin_report_edit {
 
             # If they clicked the no more updates button, we're done.
             if ($c->req->param('no_more_updates')) {
-                my $extra = $problem->extra || {};
-                $extra->{subdiv_overdue} = $self->overdue( $problem );
-                $problem->extra( $extra );
+                $problem->set_extra_metadata_if_undefined( subdiv_overdue => $self->overdue( $problem ) );
```

Comment:
The old code wasn't only if undefined (I don't know if this matters), presumably not, though I guess an edge case of a report going backwards in process then forwards again?